crates/jstzd/src/task/octez_client.rs

```diff
@@ -380,6 +386,24 @@ impl OctezClient {
         }
         Ok((contract_address.unwrap(), operation_hash.unwrap()))
     }
+
+    pub async fn transfer(
```

Comment:
FYI i'm using the transfer command in call contract too, wondering if we should just have one general function to transfer / call contract?

Reply:
sure